pcdsdevices/epics_motor.py

```diff
@@ -125,17 +127,43 @@ def limits(self, limits):
         self.low_limit = limits[0]
         self.high_limit = limits[1]
 
+    def enable(self, enable):
+        """
+        Sets the state for the motor
+
+        Parameters
+        ----------
+        enable: Bool
+
+        Returns
+        -------
+        status: Status Object
+            Status object of the set
+        """
+        return self.disabled.set(not enable)
+
     def check_value(self, value):
         """
+        Check if the motor is disabled
         Check if the value is within the soft limits of the motor.
 
         Raises
         ------
+        Exception
+            If the motor is passed any motion command when disabled
         ValueError
+            When the provided value is outside the range of the low
+            and high limits
         """
         # First check that the user has returned a valid EPICS value. It will
         # not consult the limits of the PV itself because limits=False
         super().check_value(value)
+
+        # Find the value for the disabled attribute
+        if self.disabled.value == 1:
+            raise Exception("Motor is not enabled. Motion requests "
+                            "ignored")
```

Comment:
This should be more specific than `Exception`, since every error is an `Exception`, but I'm struggling to pick a good replacement. I'm tempted to say this should be a `RuntimeError`,  but that's not much better. Any suggestions @teddyrendahl ?